studies/views/submitted_studies.py

```diff
@@ -66,10 +66,17 @@ def get_queryset(self):
         qs = super().get_queryset()
         if self.action in ["my_studies"]:
             # for my studies we need to limit that
-            qs = qs.filter(submitter=self.request.user)
+            if hasattr(self.request.user, "profile") and self.request.user.profile.is_reviewer:
+                qs = qs.filter(Q(approval_status=ApprovalChoices.PENDING) | Q(submitter=self.request.user))
```

Comment:
so if I have reviewer permissions I can see other submitters' pending studies as well as my own (which is the base case)?

Reply:
exactly!